console/console-init/ui/src/utils/utils.ts

```diff
@@ -48,4 +53,12 @@ const getSelectOptionList = (list: string[], totalRecords: number) => {
   return records;
 };
 
-export { getSelectOptionList };
+const removeForbiddenChars = (input: string) => {
+  let modifiedInput = input.replace(forbiddenBackslashRegexp, "\\\\");
+  modifiedInput = modifiedInput.replace(forbiddenSingleQuoteRegexp, "''");
```

Comment:
I think replacing ' with ' ' doesn't work in this case. 
![image](https://user-images.githubusercontent.com/29524461/78749507-0422e700-798c-11ea-8a1a-8ae56c028f84.png)